bazel/repositories.bzl

```diff
@@ -197,6 +199,14 @@ def _envoy_api_deps():
         name = "http_api_protos",
         actual = "@googleapis//:http_api_protos",
     )
+    auth_bind_targets = [
+        "auth"
+    ]
+    for t in auth_bind_targets:
```

Comment:
FWIW, the recent merges of https://github.com/envoyproxy/data-plane-api/pull/421 and https://github.com/envoyproxy/envoy/pull/2430 will break this (and other parts of your PR). Be prepared to update when you merge master or pull in an updated data-plane-api SHA.

Reply:
Thanks for the heads up. I'll do another merge with master to pull in the latest data-plane-api and rework the impacted code.